feature/experimental/isis/otg_tests/base_adjacencies_test/README.md

```diff
@@ -4,164 +4,204 @@
-
-## Protocol/RPC Parameter coverage
-
-*   IS-IS:
-    *   LSP messages
-        *   TLV 1 (Area Addresses)
-        *   TLV 10 (Authentication)
-        *   TLV 22 (Extended IS reach)
-        *   TLV 135 (Extended IP Reachability)
-        *   TLV 137 (Dynamic Name)
-        *   TLV 232 (IPv6 Reachability)
+### Test environment setup
+
+*   DUT has an ingress port and 1 egress port.
+
+    ```
+                             |         |
+        [ ATE Port 1 ] ----  |   DUT   | ---- [ ATE Port 2 ]
+                             |         |
+    ```
+
+### RT-2.1.1 Basic fields test
+
+*   Configure DUT:port1 for an IS-IS session with ATE:port1.
+*   Read back the configuration to ensure that all fields are readable and
+    have been set properly (or correctly have their default value).
+*   Check that all relevant counters are readable and are 0 since the
+    adjacency has not yet been established.
+*   Push ATE configuration for the other end of the adjacency, and wait for
+    the adjacency to form.
+*   Check that the various state fields of the adjacency are reported
+    correctly.
+*   Check that error counters are still 0 and that packet counters have all
+    increased.
+
+### RT-2.1.2 Hello padding test
+
+*   Configure IS-IS between DUT:port1 and ATE:port1 for each possible value
+    of hello padding (DISABLED, STRICT, etc.)
+*   Confirm in each case that that adjacency forms and the correct values
+    are reported back by the device.
+
+### RT-2.1.3 Authentication test
+
+*   Configure IS-IS between DUT:port1 and ATE:port1 With authentication
+    disabled, then enabled in TEXT mode, then enabled in MD5 mode.
+*   Confirm in each case that that adjacency forms and the correct values
+    are reported back by the device.
+
+### RT-2.1.4 ISIS circuit-type point-to-point
+
+*   Configuration:
+    *   Configure ISIS for ATE port-1 and DUT port-1.
+    *   Configure both DUT and ATE interfaces as ISIS type point-to-point.
+*   Verification:
+    *   Verify that ISIS adjacency is coming up.
+    *   Verify the output of streaming telemetry path displaying the interface circuit-type as point-to-point.
+
+### RT-2.1.5 Routing test
+
+*   Configure ISIS level authentication  and hello authentication.
+*   Ensure that IPv4 and IPv6 prefixes that are advertised as attached
+    prefixes within each LSP are correctly installed into the DUT
+    routing table, by ensuring that packets are received to the attached
+    prefix when forwarded from ATE port-1.
+*   Ensure that IPv4 and IPv6 prefixes that are advertised as part of an
+    (emulated) neighboring system are installed into the DUT routing
+    table, and validate that packets are sent and received to them.
+*   With a known LSP content, ensure that the telemetry received from the
+    device for the LSP matches the expected content.
+
+### RT-2.1.6 Hello timer and multiplier customization test
```

Comment:
Also add TODO link here.  (Note, the link can point to the same task/issue.  No need to create multiple issues.)